src/main/java/calculator/ExpressionValidator.java

```diff
@@ -0,0 +1,59 @@
+package calculator;
+
+public class ExpressionValidator {
+
+  public static void validate(String expression) {
+    isEmpty(expression);
+    String[] expressionElements = expression.split(" ");
```

Comment:
`" "` 구분자를  상수로 만들어 이름을 부여하면, 이후에 코드를 읽는데 더 도움이 되지 않을까요? 😄 
```suggestion
    String[] expressionElements = expression.split(DELIMITER);
```